plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -617,7 +616,6 @@ public static void validateTableCanBeDropped(Table table)
     {
         // TODO: support path override in Iceberg table creation: https://github.com/trinodb/trino/issues/8861
         if (table.properties().containsKey(OBJECT_STORE_PATH) ||
-                table.properties().containsKey(WRITE_NEW_DATA_LOCATION) ||
```

Comment:
Should we inline this string to handle old tables that still reference it?

Reply:
Sure, I can inline and add a comment for backward compatibility.